apps/docs/docs/template/api/virtual-scrolling.md

```diff
@@ -0,0 +1,862 @@
+- works also with static variables `*rxVirtualFor="let i of myData"`
+- Immutable as well as mutable data structures (`trackBy`)
+- Notify when rendering of templates is finished (`renderCallback`)
+
+**Performance Features**
+
+- lazy template creation (done by [Render Strategies](../../cdk/render-strategies/))
+- non-blocking rendering of lists [Concurrent Strategies](../../cdk/render-strategies/strategies/concurrent-strategies.md)
+- configurable frame budget (defaults to 60 FPS)
+- Super efficient layouting with css transformations
+- Scoped layouting with css containment
+- Define a viewCache in order to re-use views instead of re-creating them
+- triggers change-detection on `EmbeddedView` level
+- Zone-agnostic, opt-out of `NgZone` with `patchZone`
+- 3 Configurable `RxVirtualScrollStrategy` providing the core logic to calculate the viewRange and position DOM
+  Nodes
+  - `FixedSizeVirtualScrollStrategy`
+  - `AutosizeVirtualScrollStrategy`
+  - `DynamicSizeVirtualScrollStrategy`
+
+## Usage Examples
+
+> **⚠ Notice:**
+> By default `*rxVirtualFor` is optimized for performance out of the box.
+>
+> This includes:
+>
+> - The default render strategy is [`normal`](../../cdk/render-strategies/strategies/concurrent-strategies.md).
+>   This ensures non-blocking rendering but can cause other side-effects. See [strategy configuration](../../cdk/render-strategies#Default-configuration) if you want to change it.
+> - Creates templates lazy and manages multiple template instances
+
+### Setup
+
+You have to import 3 parts in order to get started:
+
+- the `Viewport`
+- the `ViewRepeater`
+- the `ScrollStrategy`
+
+```ts
+import {
+  FixedSizeVirtualScrollStrategy, // ScrollStrategy
+  RxVirtualScrollViewportComponent, // Viewport
+  RxVirtualFor, // ViewRepeater
+} from '@rx-angular/template/experimental/virtual-scrolling';
+```
+
+Module based setup:
+
+```
+import { ForModule } from "@rx-angular/template/for";
+
+@NgModule({
+  imports: [RxVirtualFor, RxVirtualScrollViewportComponent, FixedSizeVirtualScrollStrategy],
+})
+export class MyModule {}
+```
+
+Standalone component setup:
+
+```
+@Component({
+  standalone: true,
+  imports: [RxVirtualFor, RxVirtualScrollViewportComponent, FixedSizeVirtualScrollStrategy],
+})
+export class MyComponent {}
+```
+
+### Fixed size virtual-scroll using `*rxVirtualFor` with `Observable` values
+
+```html
+<rx-virtual-scroll-viewport>
+  <div class="hero" [itemSize]="50" *rxVirtualFor="let hero of heroes$;">
+    <div>
+      <div><strong>{{ hero.name }}</strong></div>
+      <div>{{ hero.id }}</div>
+      <div>{{ hero.description }}</div>
+    </div>
+  </div>
+</rx-virtual-scroll-viewport>
+```
+
+```scss
+.hero {
+  height: 50px; // this is important, the items have to be sized properly
+}
+```
+
+```ts
+@Component({})
+export class AnyComponent {
+  heroes$: Observable<Hero[]> = getHeroes();
+}
+```
+
+### Fixed size virtual-scroll using `*rxVirtualFor` with static values
```

Comment:
Maybe keep one and mention static values?

Reply:
We need to reference other approaches section somewhere here, otherwise it's too hidden.